spec/support/capybara.rb

```diff
@@ -5,16 +5,16 @@
 require 'selenium/webdriver'
 
 Capybara.register_driver :headless_chrome do |app|
-  browser_options = Selenium::WebDriver::Chrome::Options.new
-  browser_options.args << '--headless' if !ENV['SHOW_BROWSER']
-  browser_options.args << '--disable-gpu' if !ENV['SHOW_BROWSER']
-  browser_options.args << '--window-size=1200x700'
-  browser_options.args << '--no-sandbox'
-  browser_options.args << '--disable-dev-shm-usage'
+  options = Selenium::WebDriver::Chrome::Options.new
+  options.add_argument('--headless') if !ENV['HEADLESS_BROWSER']
+  options.add_argument('--disable-gpu') if !ENV['HEADLESS_BROWSER']
```

Comment:
Was it intended to change the env var here and in the file below?

Reply:
nope, thank you 🤦🏼‍♂️